CancerGov/_src/StyleSheets/_homelanding.scss

```diff
@@ -275,20 +276,58 @@ a.learn-more {
 	/* adding padding per #OCECREATIV-783 --sjc */
 }
 
-/* Class used to target homepage only with blue tint on 'Resources for' */
+/* Class previously used to target homepage only with blue tint on 'Resources for' */
```

Comment:
I think this comment might be confusing since it doesn't explain the current code but the previous code.

Reply:
it was left there as a reference as to why the '.guide-care--home' class was created in the first place, but it can be removed/edited.

Reply:
If the class is totally redundant then it should be removed. Which is a template change inSubLayout_Home_Guide.ascx . It seems, anytime we don't kill deprecated code at the moment of deprecation, it seems to turn into unkillable zombie code on us.

However, it's not clear to me that it is deprecated since all these changes are scoped to it. I assume guide cards on other pages are not getting the same treatment now and we do still want the class?

Reply:
Yes, we want to keep the class because all of the guide card changes (for [WCMSFEQ-1028]) are scoped to that class (homepage only). Either way, comment was updated to reflect current ticket's request.